tests/infrastructure/virt-handler.go

```diff
@@ -114,7 +114,7 @@ var _ = DescribeInfra("virt-handler", func() {
 		for _, node := range nodesToEnableKSM {
 			Eventually(func() (string, error) {
 				command := []string{"cat", "/sys/kernel/mm/ksm/run"}
-				ksmValue, err := libnode.ExecuteCommandInVirtHandlerPod(node, command)
+				ksmValue, _, err := libnode.ExecuteCommandOnNodeThroughVirtHandler(node, command)
```

Comment:
In cases like this (across the PR) you are losing the stderr which was provided in ExecuteCommandInVirtHandlerPod maybe its a relevant information for debug this tests.